### PR TITLE
Hide nav trigger checkbox.

### PR DESCRIPTION
**Pull request by DamianMullins**

### CODE REVIEW

**Comment by DamianMullins** [comment]
The is no changelog currently. It should be created as a separate task as it's quite time-consuming to back through all the commits to list the changes to each version 😓

**Comment by DamianMullins** [comment]
I've created #7 to cover the task of adding the changelog.

File: src/scss/partials/_nav.scss [hunk 1]
```diff
@@ -241,9 +241,7 @@ $nav-popover-padding          : spacing(x2);
 // Navigation Trigger
 // This is the checkbox that controls the menu active state without JS via :checked
 .c-nav-trigger {
-    position: absolute;
-    top: -100px;
-    left: -100px;
+    display: none;
```
Comment (ashleynolan) [inline src/scss/partials/_nav.scss]: Would be good to double check if this affects accessibility in some screen readers.

Most articles I’ve seen seem to advise not to display none screen reader components…

**Comment by DamianMullins** [comment]
Have chatted with @ashleynolan and will be updating this PR to change the behaviour of the menu toggle in a similar way to the Guardian menu behaviour.

**Comment by DamianMullins** [comment]
Closing. Will submit a related PR which improves accessibility.

**Pull request closed without merging**